ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocol.java

```diff
@@ -169,4 +169,8 @@ public static EthProtocol get() {
     public static final int V65 = 65;
     public static final int V66 = 66;
   }
+
+  public static boolean isEth66Compatible(final Capability capability) {
+    return capability.getName().equals(NAME) && capability.getVersion() >= ETH66.getVersion();
```

Comment:
Would `Object.equals(capability.getName(), NAME)` would be safer, as you wouldn't have to do null checks?

Reply:
Changed to do ```NAME.equals(capability.getName())```